doc/source/user/absolute_beginners.rst

```diff
@@ -5,212 +5,195 @@ NumPy: the absolute basics for beginners
-working with numerical data in Python, and it's at the core of the scientific
-Python and PyData ecosystems. NumPy users include everyone from beginning coders
-to experienced researchers doing state-of-the-art scientific and industrial
-research and development. The NumPy API is used extensively in Pandas, SciPy,
-Matplotlib, scikit-learn, scikit-image and most other data science and
-scientific Python packages.
-
-The NumPy library contains multidimensional array and matrix data structures
-(you'll find more information about this in later sections). It provides
-**ndarray**, a homogeneous n-dimensional array object, with methods to
-efficiently operate on it. NumPy can be used to perform a wide variety of
-mathematical operations on arrays.  It adds powerful data structures to Python
-that guarantee efficient calculations with arrays and matrices and it supplies
-an enormous library of high-level mathematical functions that operate on these
-arrays and matrices.
-
-Learn more about :ref:`NumPy here <whatisnumpy>`!
-
-Installing NumPy
-----------------
-
-To install NumPy, we strongly recommend using a scientific Python distribution.
-If you're looking for the full instructions for installing NumPy on your
-operating system, see `Installing NumPy <https://numpy.org/install/>`_.
-
-
-
-If you already have Python, you can install NumPy with::
-
-  conda install numpy
-
-or ::
-
-  pip install numpy
-
-If you don't have Python yet, you might want to consider using `Anaconda
-<https://www.anaconda.com/>`_. It's the easiest way to get started. The good
-thing about getting this distribution is the fact that you don’t need to worry
-too much about separately installing NumPy or any of the major packages that
-you’ll be using for your data analyses, like pandas, Scikit-Learn, etc.
+NumPy (**Num**\ erical **Py**\ thon) is an open source Python library that's
+widely used in science and engineering. The NumPy library contains
+multidimensional array data structures, such as the homogeneous, N-dimensional
+``ndarray``, and a large library of functions that operate efficiently on these
+data structures. Learn more about NumPy at :ref:`What is NumPy <whatisnumpy>`,
+and if you have comments or suggestions, please
+`reach out <https://numpy.org/community/>`_!
 
 How to import NumPy
 -------------------
 
-To access NumPy and its functions import it in your Python code like this::
+After `installing NumPy <https://numpy.org/install/>`_, it may be imported
+into Python code like::
 
   import numpy as np
 
-We shorten the imported name to ``np`` for better readability of code using
-NumPy. This is a widely adopted convention that makes your code more readable
-for everyone working on it. We recommend to always use import numpy as ``np``.
+This convention allows access to NumPy features with a short, recognizable
+prefix (``np.``) while distinguishing NumPy features from others that have the
+same name.
```

Comment:
I'd advocate for leaving some explanation that his, while not mandatory, is a widely used convention. I've seen beginners confused by this before.

Reply:
```suggestion
This widely-used convention allows access to NumPy features with a short, recognizable
prefix (``np.``) while distinguishing NumPy features from others that have the
same name.
```
or "nearly universal"?